src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseAccessService.java

```diff
@@ -68,6 +72,10 @@ public AccessLevel getAccessLevel(ServiceAndUserDetails serviceAndUserDetails) {
         return new HashSet<>(caseUserRepository.findCaseRoles(Long.valueOf(caseId), userRepository.getUserId()));
     }
 
+    public Set<String> getCaseCreationCaseRoles() {
```

Comment:
beware now this method is not only returning case roles, but also the idam roles. So we should rename it to a more generic `getCaseCreationRoles`